package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@brainhubeu/react-carousel",
-  "version": "1.4.0",
+  "version": "1.8.0",
```

Comment:
is there any reason why we should bump version to 1.8.0? Maybe bumping patch version would be enough? what do you think?

Reply:
@damian-brainhub 
it's because previous versions are already published in testing this workflow - sorry for bumping minor version instead of patch version when testing